src/en-GB/discover/technology/index.html

```diff
@@ -6,32 +6,34 @@
 <h1 class="h1">Technology</h1>
 
 <section class="bg-grey-300 section section--full">
-  <h2 class="h2 text-center">Discover more</h2>
+  <div class="wrapper">
+    <h2 class="h2 text-center">Discover more</h2>
 
-  <ul class="grid lg:grid--cols-3 list-none mb-0 p-0 ul">
-    <li class="bg-white border-grey-500 border-px border-solid p-5 relative rounded-2">
-      <img class="mb-4 w-12" src="/assets/svgs/icon-benefits-blue-red.svg" alt="" />
-      <h3 class="h3">Technology</h3>
-      <p class="p">
-        Donec dignissim vestibulum lorem, sed pulvinar erat consequat sed. Quisque elementum sem eget magna feugiat mattis.
-      </p>
-      <a class="a link-cover link-cover--shadow" href="/{{ locale }}/discover/technology/">Learn more about the technology behind Ceph</a>
-    </li>
-    <li class="bg-white border-grey-500 border-px border-solid p-5 relative rounded-2">
-      <img class="mb-4 w-12" src="/assets/svgs/icon-customers-blue-red.svg" alt="" />
-      <h3 class="h3">Customers</h3>
-      <p class="p">
-        Donec dignissim vestibulum lorem, sed pulvinar erat consequat sed. Quisque elementum sem eget magna feugiat mattis.
-      </p>
-      <a class="a link-cover link-cover--shadow" href="/{{ locale }}/discover/customers/">See who is using Ceph</a>
-    </li>
-    <li class="bg-white border-grey-500 border-px border-solid p-5 relative rounded-2">
-      <img class="mb-4 w-12" src="/assets/svgs/icon-case-study-blue-red.svg" alt="" />
-      <h3 class="h3">Solutions</h3>
-      <p class="p">
-        Donec dignissim vestibulum lorem, sed pulvinar erat consequat sed. Quisque elementum sem eget magna feugiat mattis.
-      </p>
-      <a class="a link-cover link-cover--shadow" href="/{{ locale }}/solutions/">Case studies of Ceph in action</a>
-    </li>
-  </ul>
+    <ul class="grid lg:grid--cols-3 list-none mb-0 p-0 ul">
+      <li class="bg-white border-grey-500 border-px border-solid p-5 relative rounded-2">
+        <img class="mb-4 w-12" src="/assets/svgs/icon-benefits-blue-red.svg" alt="" />
+        <h3 class="h3">Technology</h3>
+        <p class="p">
+          Donec dignissim vestibulum lorem, sed pulvinar erat consequat sed. Quisque elementum sem eget magna feugiat mattis.
+        </p>
+        <a class="a link-cover link-cover--shadow" href="/{{ locale }}/discover/technology/">Learn more about the technology behind Ceph</a>
+      </li>
```

Comment:
These are looking nice. Looks like we want to chat about componentisation fairly soon now modules like these are defined?

Reply:
Agreed.

When I start page building I'll add a few `<!-- start -->....<!-- end -->` around potential components to help rationalise them. We can also make reference to this in the README's we'll create for administering the pages.